VideoLocker/res/layout/activity_course_base.xml

```
@@ -39,8 +39,10 @@
             <LinearLayout
                 android:id="@+id/last_access_bar"
                 android:orientation="vertical"
-                style="@style/last_access_bar"
-                android:visibility="gone">
+                android:layout_width="match_parent"
+                android:layout_height="50dp"
+                android:background="@color/edx_brand_primary_x_light"
+                android:visibility="visible">
```

Shouldn't this be `gone` by default? I don't think the code hides it initially.